evergreen/evergreen.yml

```diff
@@ -537,23 +534,14 @@ axes:
         variables:
           DRIVER_VERSION: "latest"
 
-  - id: target_runtime
```

Comment:
Decided to remove it for now. As we have a single target framework in the project.